src/_langs/en/tools/javascript/console/structured-data.markdown

```diff
@@ -15,20 +15,54 @@ priority: 0
 collection: console
 key-takeaways:
   tldr-tbd:
-    - TBD tldr.
+    - Compare similar data objects using `table()`.
+    - Build an array to define property strings
 remember:
   note-tbd:
     - TBD note.
 ---
 {% wrap content %}
 
-TBD. Doc: https://developer.chrome.com/devtools/docs/console#viewing-structured-data
-
 {% include modules/takeaway.liquid list=page.key-takeaways.tldr-tbd %}
 
-### TBD
+### Viewing Structured Data
+
+The `table()` method provides an easy way to view similar data objects. This will take the properties of an object and create a header. Row data comes from each indexes' properties value.
+
+#### A table in the console using two arrays.
+
+Example code:
+
+    console.table([{a:1, b:2, c:3}, {a:"foo", b:false, c:undefined}]);
+    console.table([[1,2,3], [2,3,4]]);
+  
+The output of the example code:
+
+<img src="imgs/table-arrays.png" class="center" alt="console table display"/>
+
+The second parameter to table() is optional. You may define an array containing the property strings you wish to display.
+
+#### A console table using a collection of objects.
+
+Example code:
+
+    function Person(firstName, lastName, age) {
+      this.firstName = firstName;
+      this.lastName = lastName;
+      this.age = age;
+    }
+    
+    var family = {};
+    family.mother = new Person("Susan", "Doyle", 32);
+    family.father = new Person("John", "Doyle", 33);
+    family.daughter = new Person("Lily", "Doyle", 5);
+    family.son = new Person("Mike", "Doyle", 8);
+  
+    console.table(family, ["firstName", "lastName", "age"]);
+
+The output of the example code:
 
-TBD.
+<img src="imgs/table-people-objects.png" class="center" alt="console output with table objects"/>
```

Comment:
This needs to change to markdown format ![](). And should map to the image that you've added, which I think is images folder?